snapshots/windows/windows.go

```diff
@@ -457,12 +488,47 @@ func (s *snapshotter) createUVMScratchLayer(ctx context.Context, snDir string, p
 		return errors.Wrapf(err, "failed to find UtilityVM directory in base layer %q", baseLayer)
 	}
 
-	templateDiffDisk := filepath.Join(uvmPath, "SystemTemplate.vhdx")
+	var (
+		newDisks         = sizeGB > 0 && sizeGB != 10
```

Comment:
I think just `sizeGB != 10` should be enough because if sizeGB is 0 we will set it to `10` above anyway.

Reply:
Good catch